ui/src/styles.css

```diff
@@ -1,3 +1,135 @@
+a:link {
+  color: #0079b8;
+  text-decoration: none;
+}
+
+img {
+  border-style: none;
+}
+
+hr {
+  box-sizing: content-box;
+  height: 0;
+  overflow: visible;
+}
+
+svg:not(:root) {
+  overflow: hidden;
+}
+
+button,
+input,
+select,
+textarea {
+  font: inherit;
+  margin: 0;
+}
+
+button,
+html [type="button"],
+[type="reset"],
+[type="submit"] {
+  -webkit-appearance: button;
+}
+
+button,
+input {
+  overflow: visible;
+}
+
+button,
+select {
+  text-transform: none;
+}
+
+button:focus {
+  text-decoration: none;
+  outline: none;
+  border: none;
+  box-shadow: none;
+  background: none;
+}
+
+fieldset {
+  border: 1px solid #c0c0c0;
+  margin: 0 2px;
+  padding: 0.35em 0.625em 0.75em;
+}
+
+legend {
+  box-sizing: border-box;
+  color: inherit;
+  display: table;
+  max-width: 100%;
+  padding: 0;
+  white-space: normal;
+}
+
+section {
+  display: block;
+}
+
+h1,
+h2,
+h3,
+h4 {
```

Comment:
```suggestion
h1, h2, h3, h4 {
```

Reply:
Codacity was giving me an error when I did it this way